### PR TITLE
TEET-1336 delete cooperation application response

### PR DESCRIPTION
**Pull request by tatut**

### CODE REVIEW

File: app/frontend/src/cljs/teet/cooperation/cooperation_view.cljs [hunk 1]
```diff
@@ -662,6 +662,19 @@
                                   (fn [e!]
                                     (e! (close-event))
                                     (e! (cooperation-controller/->ResponseCreated response (boolean (:db/id @form-atom)))))))
+                 :delete (when (:db/id @form-atom)
```
Comment (jkorpelin) [inline app/frontend/src/cljs/teet/cooperation/cooperation_view.cljs:665]: Does this need some authorization checking? Or should the defcommands pre check be used here to show a disabled delete button when an opinion exists?

Comment (tatut) [reply app/frontend/src/cljs/teet/cooperation/cooperation_view.cljs:665]: If you can create/edit, then you can also delete afaict

**Review by jkorpelin — APPROVED** [review]
If the situations in the comment are taken care of somewhere else this looks good.

File: app/frontend/src/cljs/teet/cooperation/cooperation_view.cljs (hunk 1 above)
Comment (tatut) [reply app/frontend/src/cljs/teet/cooperation/cooperation_view.cljs:665]: The pre check is already checked, the "edit" button isn't shown if the application has an opinion

**Pull request merged**